src/main/resources/templates/email/placement-updated-week-12/v1.0.0.html

```diff
@@ -6,35 +6,37 @@
 <body>
 <h1>Email Message</h1>
 <h2>Subject</h2>
-<th:block th:fragment="subject">Placement confirmation</th:block>
+<th:block th:fragment="subject">NHS England - <th:block th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : _"></th:block> - Notification of Placement</th:block>
 <h2>Content</h2>
 <th:block th:fragment="content">
   <p th:if="${#strings.isEmpty(givenName) AND #strings.isEmpty(familyName)}">Dear Doctor,</p>
   <p th:if="${not (#strings.isEmpty(givenName) OR #strings.isEmpty(familyName))}">Dear <span th:text="${not #strings.isEmpty(title)} ? ${title} : 'Dr'"></span> <span th:text="${not #strings.isEmpty(givenName)} ? ${givenName} : _"></span> <span th:text="${not #strings.isEmpty(familyName)} ? ${familyName} : _"></span>,</p>
-  <p th:if="${not #strings.isEmpty(gmcNumber)}">GMC <span th:text="${gmcNumber}"></span></p>
+  <p>GMC <th:block th:text="${not #strings.isEmpty(gmcNumber)} ? ${gmcNumber} : _">UNKNOWN</th:block><th:block th:if="${#strings.isEmpty(isValidGmc) OR not isValidGmc}"> - We don't have a confirmed GMC no. please contact your local office to update this.</th:block></p>
+  <p>
+    This notification pertains to your placement in <b><th:block th:text="${not #strings.isEmpty(specialty)} ? ${specialty} : _">(speciality name missing)</th:block></b> at <b><th:block th:text="${not #strings.isEmpty(site)} ? ${site} : _">(site name missing)</th:block></b> starting on <b><th:block th:text="${startDate} ? |${#temporals.format(startDate, 'dd MMMM yyyy')}| : _">(start date missing)</th:block></b>.
+  </p>
   <p>
     We appreciate this must be a busy time for you but please set some time aside to read this letter as it contains some vital information regarding your training placement.
   </p>
   <p>
-    The NHS England team can now confirm your placement within a <b><span th:text="${localOfficeName}"></span></b> programme.
+    The NHS England team can now confirm your placement within a <b><span th:text="${localOfficeName}"></span></b> programme. {Please note Healthcare Education England is now part of NHS England. Our systems will be updated in due course}
```

Comment:
Maybe include the default if localOfficeName is missing. I think the '{' and '}' were not intended to be included, btw :)